src/clone.rs

```diff
@@ -420,6 +426,9 @@ macro_rules! clone {
     ($($(@ $strength:ident)? $($variables:ident).+ $(as $rename:ident)?),+ => async $($x:tt)+ ) => (
         compile_error!("async blocks are not supported by the clone! macro");
     );
+    ($($(@ $strength:ident)? $variables:expr),+ => move || $($_:tt)* ) => (
```

Comment:
And for this?

Reply:
I added a test for it (the "default-return" one).